corehq/apps/locations/tests/test_change_has_user.py

```diff
@@ -66,6 +67,7 @@ def submit_form(self, name):
         self.assertTrue(form.is_valid())
         form.save()
 
+    @flaky
```

Comment:
What does `@flaky` do?  Has this test been intermittently failing recently?

Reply:
By default, it re-attempts the test a second time and passes the build if that second attempt succeeds. https://pypi.org/project/flaky/

It's bitten me a few times in the past week. I don't know what our threshold should be for adding the decorator, open to suggestions.

Reply:
huh, cool - I hadn't seen it before, looks like it was just added?  Seems like a reasonable approach.  I do hate essentially admitting defeat with flaky tests, but this seems vastly better than waiting for a failure and rebuliding the whole job, and better than requiring PR authors to fix unrelated test failures.

Reply:
I definitely share your ambivalence.